libbeat/cmd/instance/beat.go

```diff
@@ -531,7 +531,7 @@ func (b *Beat) Setup(settings Settings, bt beat.Creator, setup SetupSettings) er
 			if outCfg.Name() != "elasticsearch" {
 				return fmt.Errorf("Index management requested but the Elasticsearch output is not configured/enabled")
 			}
-			esClient, err := eslegclient.NewConnectedClient(outCfg.Config())
+			esClient, err := eslegclient.NewConnectedClient(outCfg.Config(), b.Info.Name)
```

Comment:
this can be set to hostname in some cases, is that ok?

Reply:
that's not really what we want, is there a way we can get the `*beat` name (metricebeat, heartbet, etc)?

Reply:
i think this one will be correct most of the time but i've seen some initialization with hostname. would be good to get some input from somebody who knows this a bit better

Reply:
The beatname is under `b.Info.Beat`. `b.Info.Name` is sometimes the hostname, or whatever the user has set in `name` in the config file.